src/Core/Enums/SupportedDatabaseProviders.cs

```diff
@@ -0,0 +1,10 @@
+namespace Bit.Core.Enums
+{
+    public enum SupportedDatabaseProviders 
+    {
+        MsSql,
```

Comment:
Maybe call it SqlServer, which matches the repo namespace? To me, this always ends up looking too close to MySql when reading